<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicio</class>
 <widget class="QDialog" name="inicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio Snake 2.0</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>177</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>BIENVENIDO A SNAKE 2.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>135</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>¿Qué quieres hacer?</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>112</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_nuevo">
        <property name="text">
         <string>Nueva partida</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cargar">
        <property name="text">
         <string>Cargar Partida</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_salir">
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
